moduleroot/Gemfile

```diff
@@ -92,7 +94,8 @@ group <%= group %> do
   ", :git => '#{gem['git']}'" if gem['git'] %><%=
   ", :branch => '#{gem['branch']}'" if gem['branch'] %><%=
   ", :ref => '#{gem['ref']}'" if gem['ref'] %><%=
-  ", :platforms => '#{gem['platforms']}'" if gem['platforms'] %><%=
+  ", :platforms => #{gem['platforms']}" if gem['platforms'].is_a?(Array) %><%=
+  ", :platforms => \"#{gem['platforms']}\"" if gem['platforms'] && !gem['platforms'].is_a?(Array) %><%=
```

Comment:
I have needed to make this change because supplying multiple platforms for a gem dependency generated a gemfile with `platforms => '["mswin","mingw","x64_mingw"]'` rather than `platforms => ["mswin","mingw","x64_mingw"]` without the quotes surrounding the array.